puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release_nightly_test_deb.yaml

```diff
@@ -1,28 +1,38 @@
 - job:
     name: release_nightly_test_deb
+    node: 'el && ipv6'
     project-type: matrix
     block-downstream: true
     block-upstream: true
     quiet-period: 30
     logrotate:
-      daysToKeep: 2
-      numToKeep: -1
+      daysToKeep: 8
+    scm:
+      - forklift
     triggers:
       - timed: 'H 07 * * 1-5'
-    builders:
-      - trigger-builds:
-        - project: systest_foreman
-          predefined-parameters: "os=${os}\nrepo=nightly\nstaging=true"
-          block: true
     axes:
       - axis:
           type: user-defined
           name: os
           values:
           - debian9
           - ubuntu1604
+    builders:
+      - systest:
+          repo: nightly
+          repo_environment: release
```

Comment:
I think this should be staging.

Reply:
@ekohl `release` is correct. release_nightly_test_deb currently calls systest_foreman with a `staging=true` variable, but that parameter is not even defined on systest_foreman.

Reply:
I talked to @mmoll and I'm pretty sure that's actually a regression I introduced when refactoring. I still needed to find exactly where, but it was causing issues in the Debian pipeline.

Reply:
I'll do just this one because we're friends